### PR TITLE
Bookstore: use Redux in React components.

### PR DESCRIPTION
**Pull request by a-laarabi**
# What?
In this pull request, I used the Redux store to display books. I also added some UI elements to my components and make them dispatch actions.

### Highlights:
- Understand the concept of state management.
- Use store, actions and reducers in React.
- Connect React and Redux.

Cheer and happy reviewing 🥇

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @a-laarabi 🙋‍♀️,  

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request
 
You have submitted a project with a README that is not documented. Please document your README using the Microverse template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
 import App from './App';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <BrowserRouter>
-      <App />
+      <Provider store={store}>
+        <App />
+      </Provider>
```
Comment (mateo951) [inline src/index.js:15]: Main React component is wrapped in Provider and assigned store ✔

File: src/pages/Books.js [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-    { id: 1, title: 'Influence & manipulation', author: 'Robert Cialdini' },
-    { id: 2, title: '7 secrets of warren buffett', author: 'Mary Buffett' },
-  ];
+  const books = useSelector((state) => state.books);
 
   return (
     <>
       {books.map((book) => (
-        <Element key={book.id} title={book.title} author={book.author} />
+        <Element key={book.id} id={book.id} title={book.title} author={book.author} />
```
Comment (mateo951) [inline src/pages/Books.js:12]: List of books displays books from Redux store by using `useSelector` function 🙌

**Review by mateo951 — APPROVED** [review]
Hi @a-laarabi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**